internal/component/loki/source/gcplog/internal/gcplogtarget/push_translation.go

```diff
@@ -22,24 +22,28 @@ import (
 // pipeline stages
 const ReservedLabelTenantID = "__tenant_id__"
 
-// PushMessage is the POST body format sent by GCP PubSub push subscriptions.
+// PushMessageBody is the POST body format sent by GCP PubSub push subscriptions.
 // See https://cloud.google.com/pubsub/docs/push for details.
+type PushMessageBody struct {
+	Message      PushMessage `json:"message"`
+	Subscription string      `json:"subscription"`
+}
+
 type PushMessage struct {
-	Message struct {
-		Attributes       map[string]string `json:"attributes"`
-		Data             string            `json:"data"`
-		ID               string            `json:"message_id"`
-		PublishTimestamp string            `json:"publish_time"`
-	} `json:"message"`
-	Subscription string `json:"subscription"`
+	Attributes                 map[string]string `json:"attributes"`
+	Data                       string            `json:"data"`
+	MessageID                  string            `json:"messageId"`
+	DeprecatedMessageID        string            `json:"message_id"`
+	PublishTimestamp           string            `json:"publishTime"`
+	DeprecatedPublishTimestamp string            `json:"publish_time"`
```

Comment:
We are not using these fields. Should we just remove them or keep in case we want to use them in the future?

Reply:
Perhaps remove them with a comment that they are available? We could potentially use it to expose a latency metric for publish -> receive time but not sure that's really needed since GCP exposes this latency as well (https://cloud.google.com/monitoring/api/metrics_gcp_p_z#gcp-pubsub), 
<img width="855" height="206" alt="image" src="https://github.com/user-attachments/assets/6885d579-5394-4124-804a-5e80aaf0d776" />